crates/bitwarden/src/vault/cipher/cipher.rs

```diff
@@ -119,3 +123,34 @@ pub struct CipherListView {
     pub deleted_date: Option<DateTime<Utc>>,
     pub revision_date: DateTime<Utc>,
 }
+
+impl Encryptable<Cipher> for CipherView {
+    fn encrypt(self, enc: &EncryptionSettings, _: &Option<Uuid>) -> Result<Cipher> {
+        let org_id = &self.organization_id;
+        Ok(Cipher {
+            id: self.id,
+            organization_id: self.organization_id,
+            folder_id: self.folder_id,
+            collection_ids: self.collection_ids,
+            name: self.name.encrypt(enc, org_id)?,
+            notes: self.notes.encrypt(enc, org_id)?,
+            r#type: self.r#type,
+            login: self.login.encrypt(enc, org_id)?,
+            identity: self.identity.encrypt(enc, org_id)?,
+            card: self.card.encrypt(enc, org_id)?,
```

Comment:
What happens if we get null here?

Reply:
Option<T> implements Encryptable and Decryptable if T implements it, and just maps over the optional, so null stays null, and any other values get encrypted or decrypted as usual.

https://github.com/bitwarden/sdk/blob/4ae062e20e6d6bee2abd89af9dda105d19269bdc/crates/bitwarden/src/crypto.rs#L389-L399